sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -820,13 +820,31 @@ public void testCreateTableGeneratedColumnWithType() throws Exception {
     }
 
     @Test
+    public void testCreateTableGeneratedColumnWithCast() throws Exception {
+        CreateTableAnalyzedStatement analysis = (CreateTableAnalyzedStatement)analyze(
```

Comment:
isn't this test redundant with the above test? dedicated test looks better, so maybe remove the partial cast test above...